packages/admin/src/table/filterbar/difference.ts

```diff
@@ -0,0 +1,20 @@
+// inspired by: https://gist.github.com/Yimiprod/7ee176597fef230d1451#gistcomment-2565071
+
+import { isEqual, isObject, transform } from "lodash";
+
+/**
+ * Deep diff between two object, using lodash
+ * @param  {Object} object Object compared
+ * @param  {Object} base   Object to compare with
+ * @return {Object}        Return a new object who represent the diff
+ */
```

Comment:
you are using JSDoc here, which we do not use anywhere.

a single comment with:
> Returns a new object representing the diff between two objects using lodash

would be enough

Reply:
i copied the whole gist from the URL above :) didnt check about the JSDoc usage here :) i will remove it though :)